scripts/install-server.sh

```diff
@@ -4,9 +4,9 @@ set -e
 
 readonly ARGS=("$@")
 readonly PROGNAME="./install-server.sh"
-TRENTO_SERVER_CHART_VERSION=${TRENTO_SERVER_CHART_VERSION:-"1.0.0"}
-TRENTO_WEB_VERSION=${TRENTO_WEB_VERSION:-"1.0.0"}
-TRENTO_RUNNER_VERSION=${TRENTO_RUNNER_VERSION:-"1.0.0"}
+TRENTO_SERVER_CHART_VERSION=${TRENTO_SERVER_CHART_VERSION:-"1.1.0"}
+TRENTO_WEB_VERSION=${TRENTO_WEB_VERSION:-"1.1.0"}
+TRENTO_RUNNER_VERSION=${TRENTO_RUNNER_VERSION:-"1.0.1"}
```

Comment:
I am a bit confused by the runner being 1.0.1 instead of 1.1.0 :sweat_smile: 
Yet, I am probably missing some detail :innocent:, please help me remember eheh
Is it just because there were not enough new features in the runner to bump a minor? Or something else?

Reply:
Yes, that's it.
In the web and agent we have introduced new features and improvement.
The runner has none of them. Simply, super minor bugfixes

Reply:
Alright, got it! Thanks!